css-syntax-3/ident-three-code-points.html

```diff
@@ -0,0 +1,115 @@
+<!doctype html>
+<script src='/resources/testharness.js'></script>
+<script src='/resources/testharnessreport.js'></script>
+<title>Testing valid ident based on first three code points</title>
+<link rel='stylesheet' href='/resources/testharness.css'>
+<link rel='stylesheet' href='../base.css' />
+
+<link rel="author" title="Greg Whitworth" href="gwhit@microsoft.com" />
+<link rel="help" href="https://drafts.csswg.org/css-syntax-3/#would-start-an-identifier" />
+<style>
+    main div {
+      background-color:red;
+    }
+
+    #1 {
+      background-color:green;
+    }
+
+    #-2 {
+      background-color:green;
+    }
+
+    #--3 {
+      background-color:green;
+    }
+
+    #---4 {
+      background-color:green;
+    }
+
+    #a {
+      background-color:green;
+    }
+
+    #-b {
+      background-color:green;
+    }
+
+    #--c {
+      background-color:green;
+    }
+
+    #---d {
+      background-color:green;
+    }
+</style>
+<body>
+    <main>
+        <div id="1" class="item" name="one">test1</div>
+        <div id="-2" class="item" name="two">test2</div>
+        <div id="--3" class="item" name="three">test3</div>
+        <div id="---4" class="item" name="four">test4</div>
+
+        <div id="a" class="item" name="five">test A</div>
+        <div id="-b" class="item" name="six">test B</div>
+        <div id="--c" class="item" name="seven">test C</div>
+        <div id="---d" class="item" name="eight">test D</div>
+    </main>
+
+    <script>
+        var items = document.getElementsByClassName('item');
+
+        generate_tests(assert_equals, [
+            [
+                // One
+                items[0].getAttribute('name') + " should be red",

```

Comment:
Red is usually used to indicate failure. Please change the test so that all expected results are green. (This requires separating markup the elements where the ID selector is expected to match from those where it’s not.)